packages/core/src/Footer/Footer.tsx

```diff
@@ -29,8 +29,8 @@ export interface HvFooterProps extends HvBaseProps {
  */
 export const HvFooter = (props: HvFooterProps) => {
   const {
-    name = "Hitachi Vantara",
-    copyright = `© Hitachi Vantara Corporation ${new Date().getFullYear()}. All Rights Reserved.`,
+    name = "Pentaho+",
+    copyright = `© Hitachi Vantara LLC ${new Date().getFullYear()}. All Rights Reserved.`,
```

Comment:
if we're changing to LLC, shouldn't we change it elsewhere too?

Reply:
I'll update.